geode-redis/src/integrationTest/java/org/apache/geode/redis/internal/RedisStatsIntegrationTest.java

```diff
@@ -474,27 +474,27 @@ public void NetworkKiloBytesReadOverLastSecond_shouldReturnCorrectData() {
   @Test
   public void clientsStat_withConnectAndClose_isCorrect() {
 
-    jedis = new Jedis("localhost", server.getPort(), TIMEOUT);
-    jedis.ping();
+    Jedis jedis2 = new Jedis("localhost", server.getPort(), TIMEOUT);
```

Comment:
What is this change for?

Reply:
While testing, I noticed that there was a problem with the stat for `connected clients`. Occasionally this test and the next one would fail. My fix seems to have addressed the issue.

Reply:
I doubt that you ran it enough times, since this is clearly not the correct fix. The real problem is that when the connection is `closed` the stat is updated asynchronously.

Reply:
I am not sure I follow. I've run with this fix a bunch of times on my laptop and it always passes.

Reply:
And I am not sure how to make this any clearer. That is passes on your laptop means nothing! In fact you made it worse!

At least now we know that the Java Concurrency in Practice book club, you are in,  clearly is not working either.

Reply:
Maybe instead of making condescending comments, you could point me at a failing test and prove your point. No wonder everything on our team takes 3x longer than it should with people like you gumming up the works.

This is an improvement and we should finally merge it.